### DISCUSSION TITLE
Calling Postgres stored functions (with return values) and stored procs

### DISCUSSION THREAD

**Discussion by mikcatta** [jackc/pgx#1736] [Q&A]
Is there a simple example of how-to call Postgres stored func/procedures with Pgx. I am a bit new to Go and therefore Pgx, so not being lazy, just trying to understand if I can shift the backend code to Go for a new project.

**Comment by jackc** [comment]
There's nothing special about how you call server side functions or procedures with pgx. You would use `Query`, `QueryRow`, or `Exec` like you would for ordinary SQL. And the SQL would be the same you would use to call your function from psql.

e.g. `select * from my_func()`.
